domain/domain.go

```diff
@@ -323,18 +318,24 @@ func (do *Domain) loadSchemaInLoop(lease time.Duration) {
 	for {
 		select {
 		case <-ticker.C:
-			err := do.MustReload()
-			if err != nil {
+			// If it fails to reload schema,
+			// we want to try again immediately and don't need to wait for a lease.
+			for {
+				err := do.MustReload()
+				if err == nil {
+					break
+				}
 				log.Errorf("[ddl] reload schema in loop err %v", errors.ErrorStack(err))
+				time.Sleep(mustReloadSleepTime)
```

Comment:
after the `time.Sleep`, it would miss the `do.leaseCh` signal for a while, would this cause problem?